changelog.d/7567.misc

```diff
@@ -0,0 +1 @@
+Improve performance of `_get_state_groups_from_groups_txn`.
```

Comment:
```suggestion
Improve query performance for fetching state from a PostgreSQL database.
```

Ideally we like to make the changelog understandable to server admins when we can.